app/client/components/user/EditUserForm.tsx

```diff
@@ -264,13 +267,34 @@ class EditUserForm extends React.Component<
                 </SubRow>
               </Row>
               <Row>
-                <SubRow>
+                <SubRow
+                  style={{
+                    color:
+                      selectedUser.transfer_card &&
+                      selectedUser.transfer_card.is_disabled
+                        ? "#c53631"
+                        : "#555"
+                  }}
+                >
                   <InputField name="publicSerialNumber" label={"ID Number"}>
-                    {/*
-                  // @ts-ignore */}
-                    <QrReadingModal
-                      updateData={(data: string) => this.setSerialNumber(data)}
-                    />
+                    {/* // @ts-ignore */}
+
+                    <div style={{ display: "flex", alignItems: "flex-end" }}>
+                      <QrReadingModal
+                        updateData={(data: string) =>
+                          this.setSerialNumber(data)
+                        }
+                      />
+                      {selectedUser.public_serial_number ? (
```

Comment:
Should there be some visual cue that the card is already disabled?

Reply:
There is already - turns red (see line 275 above)

Reply:
Darn this colourblindness! 👀